package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pelias-wof-admin-lookup",
-  "version": "1.1.2",
```

Comment:
Minor tip: when incrementing version numbers, if you do it in a separate commit then anyone scanning a list of commits, whether on their local machine, on github, or in emails like the ones Greenkeeper sends us about what's changed in a module we require, will easily be able to see where each version change happened.